src/main.rs

```diff
@@ -152,6 +150,15 @@ fn check_apps_and_kill(
         }
     }
 }
+fn check_args(user_args: &str, args: &str) -> bool {
+    if user_args.contains("*") {
+        Regex::new(&user_args.replace("*", ".+"))
```

Comment:
Building Regex in hotpath is not a good idea, can you please move the regexes out of loops or use lazy_statics for it?
https://docs.rs/regex/latest/regex/#avoid-re-compiling-regexes-especially-in-a-loop

Reply:
sure, but we aren't going to kill it (building regex) in each iteration

Reply:
It doesn't  matter what we do to it, calling `Regex::new` in every call of `check_args` function will have expensive overhead cause we are building the regex every time, whereas we just need the `match` method to be called.

Reply:
yeah, I didn't catch what you meant at first thanks for review